src/app/course/event/course-event-create/course-event-create-edit.component.html

```diff
@@ -144,7 +144,7 @@ <h3 class="tui-form__header">
         </div>
         <div class="tui-form__buttons">
             <button
-                [disabled]="formData.invalid"
+                [disabled]="formData.invalid || clicked"
```

Comment:
Let's add a loading state with a spinner instead of disabled.
https://taiga-ui.dev/components/button